File: SumOfIntegersFromFile/cpp/SumOfIntegersFromFile.cpp
// Sum Of Integers From File program written in C++ for CodeEval.com by Steven A. Dunn

#include <cstdlib>
#include <fstream>
#include <iostream>
#include <string>

using namespace std;

int main (int argc, char* argv[])
{
	ifstream inputFile (argv[1]);
	string line;
	int sum = 0;
	
	if (inputFile)
		while (getline(inputFile, line)) { sum += atoi(line.c_str()); }
	cout << sum << endl;		
}

File: SumOfIntegersFromFile/c#/SumOfIntegersFromFile.cs
// sum of integers from file soln in c# for code eval by steven a dunn

using System;
using System.IO;

class SumOfIntegersFromFile {
	public static void Main(string[] args) {
		int sum = 0;
		using (StreamReader reader = new StreamReader(args[0])) {
			while (!reader.EndOfStream) {
				string line = reader.ReadLine();
				if (null == line)
					continue;

				sum += Int32.Parse(line);
			}
		}

		Console.WriteLine(sum);
	}
}